general-superstaq/general_superstaq/check/check_utils.py

```diff
@@ -257,7 +257,7 @@ def get_file_parser() -> argparse.ArgumentParser:
     )
     parser.add_argument("-i", "--incremental", dest="revisions", nargs="*", help=help_text)
     parser.add_argument(
-        "-x", "--exclude", action="extend", metavar="GLOB", help="Exclude files matching GLOB."
+        "-x", "--exclude", action="append", metavar="GLOB", help="Exclude files matching GLOB."
```

Comment:
ah good catch. out of curiosity would this also work?
```suggestion
        "-x", "--exclude", action="extend", nargs="+", metavar="GLOB", help="Exclude files matching GLOB."
```
if so it would allow us to use e.g. `-x file1 file2 ...` or even `-x cirq-superstaq/job*` instead of having to add a separate entry for each file

Reply:
Yep, that works!

Reply:
nice!